nss-day-cohort-45/nutyshell-leaving-the-lights-on-1
# updated jsondatabase issues

# Description
the ability to add friends from the chat is working. All friends will show up on the dom. Need to edit so  that you can not add yourself and clear up code- sending so that divs etc is available for CSS needs.
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Verify that Chat, add friends and showing all friends is working. Delete button not yet functioning.
# Checklist:
- [ x] I have performed a self-review of my own code
- [x ] My changes generate no new warnings or errors
